js/app.js

```diff
@@ -2,7 +2,7 @@
 $(document).ready(function() {
 	if (OC.currentUser !== false) {
 		$.getJSON("/index.php/apps/canary", function(data) {
-			if (data["is_adopter"]) {
+			if (data["reload"]) {
```

Comment:
Keep still the is_adopter check for the transition, should not harm

Reply:
if (is_adopter OR reload) ...

Reply:
done